specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/backuprestore.json

```diff
@@ -0,0 +1,286 @@
+              "$ref": "#/definitions/RestoreOperationParameters"
+            },
+            "description": "Azure blob SAS token to the folder where the previous successful full backup was stored"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Started full backup restore",
+            "schema": {
+              "$ref": "#/definitions/FullRestoreOperation"
+            }
+          },
+          "default": {
+            "description": "Key Vault error response describing why the operation failed.",
+            "schema": {
+              "$ref": "common.json#/definitions/KeyVaultError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create full restore from backup": {
+            "$ref": "./examples/FullRestore-example.json"
+          }
+        }
+      }
+    },
+    "/restore/{jobId}/pending": {
+      "get": {
+        "tags": [
+          "FullRestore"
+        ],
+        "operationId": "FullRestoreStatus",
+        "description": "Returns the full restore operation status from a user provided valid backup",
+        "parameters": [
+          {
+            "name": "jobId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Job Id returned part of the full restore operation"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns the full restore status",
+            "schema": {
+              "$ref": "#/definitions/FullRestoreOperation"
+            }
+          },
+          "default": {
+            "description": "Key Vault error response describing why the operation failed.",
+            "schema": {
+              "$ref": "common.json#/definitions/KeyVaultError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Full restore status": {
+            "$ref": "./examples/FullRestore-pending-example.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "RestoreOperationParameters": {
+      "properties": {
+        "sasTokenParameters": {
+          "$ref": "#/definitions/SASTokenParameter"
+        },
+        "folderToRestore": {
+          "type": "string",
+          "description": "Folder name of the blob where the previous successful backup was stored"
+        }
+      },
+      "required": [
+        "folderToRestore",
+        "sasTokenParameters"
+      ]
+    },
+    "SASTokenParameter": {
+      "properties": {
+        "storageResourceUri": {
+          "type": "string",
+          "description": "Azure blob storage container uri"
+        },
+        "token": {
+          "type": "string",
+          "description": "Azure Blob Storage SAS token pointing to the azure blob storage container"
```

Comment:
In "Azure Blob storage", 's' is always lower case. See here: https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction#about-blob-storage